app/Filament/Pages/ResourceAuditLog.php

```diff
@@ -127,11 +127,37 @@ protected function transformFieldValues(array $values): array
             if ($fieldValueMap->has($key) && is_callable($fieldValueMap->get($key))) {
                 $values[$key] = $fieldValueMap->get($key)($value);
             }
+
+            if ($this->getIsImageField($key) && is_string($value)) {
+                $values[$key] = $this->getImageUrl($key, $value);
+            }
         }
 
         return $values;
     }
 
+    protected function getIsImageField(string $fieldName): bool
+    {
+        return in_array($fieldName, [
+            'BadgeName',
+            'ImageIcon',
```

Comment:
We aren't currently tracking changes to Title/InGame/BoxArt. I'm assuming those would be added here at that point?

Reply:
They probably should be, but could need some more specialized CSS concerns given how variable the dimensions on those assets tend to be. BadgeName and ImageIcon felt like easier & valuable targets to pick off for the time being.